extensions/configuration-editing/schemas/devContainer.schema.src.json

```diff
@@ -253,51 +253,101 @@
 				"onCreateCommand": {
 					"type": [
 						"string",
-						"array"
+						"array",
+						"object"
 					],
 					"description": "A command to run when creating the container. This command is run after \"initializeCommand\" and before \"updateContentCommand\". If this is a single string, it will be run in a shell. If this is an array of strings, it will be run as a single command without shell.",
```

Comment:
Is there a short way to describe the additional option in the description? Maybe "Alternatively an object with the property keys as labels and the values as either strings or arrays of strings can be used to configure multiple commands."? (We reuse the wording of the description for the other commands.)

Reply:
Good call, updated 👍